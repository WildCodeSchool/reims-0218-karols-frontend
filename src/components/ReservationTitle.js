import React from "react"

const ReservationTitle = () => {
  return (
    <div>
      <hr />
      <h2 className="display-4">Etape 2: Créez votre réservation</h2>
    </div>
  )
}

export default ReservationTitle
