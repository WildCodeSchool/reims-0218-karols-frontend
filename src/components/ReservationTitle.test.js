import React from "react"
import ReactDOM from "react-dom"
import ReservationTitle from "./ReservationTitle"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ReservationTitle />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ReservationTitle />).toJSON()
  expect(tree).toMatchSnapshot()
})
