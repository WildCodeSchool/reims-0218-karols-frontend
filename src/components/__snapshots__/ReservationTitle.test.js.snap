// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <hr />
  <h2
    className="display-4"
  >
    Etape 2: Créez votre réservation
  </h2>
</div>
`;
